src/logic.js

```diff
@@ -265,6 +264,21 @@ define([
             });
             return messages;
         },
+        addReferences: function (mug, property, value) {
+            // get absolute paths from mug property's value
+            var isLabel = false,
```

Comment:
Would this work? `var isLabel = (!value && mug.p[property] instanceof jr.ItextItem),`